### PR TITLE
Fix 604 c

### PR DESCRIPTION
**Pull request by huard**
PR pour que Pascal puisse réviser juste ce morceau sans les autres trucs déjà révisés.

### CODE REVIEW

File: xclim/locales/fr.json [hunk 1]
```diff
@@ -727,6 +727,12 @@
     "title": "Date du fin du couvert de neige",
     "abstract": "Première date à partir de laquelle la profondeur de neige est inférieure à un seuil pendant un nombre de jours consécutifs suite à l'établissement du couvert de neige."
   },
+  "SND_MAX_DOY": {
+    "long_name": "Date de la profondeur de neige maximale",
+    "description": "Date {freq:m} à laquelle la profondeur de neige atteint sa valeur maximale.",
+    "title": "Date où la profondeur de neige est maximale",
+    "abstract": "Jour de l'année où la profondeur de neige atteint sa valeur maximale."
```
Comment (aulemahal) [inline xclim/locales/fr.json:734]: Il me semble que la meilleure traduction serait "épaisseur" de neige, non?
Qu'en pense @sbiner?

**Review by aulemahal — APPROVED** [review]
Ben parf.
This way of masking according to a condition is elegant!

**Comment by aulemahal** [comment]
@Zeitsperre  I just relauched the github checks. They failed upon loading a dataset from xclim-testdata with HTTP error 500. It's not the first time I see this. Last time, rerunning the jobs did the trick. Have you seen that elsewhere, where we use the open-dataset-from-github trick?

**Comment by Zeitsperre** [comment]
@aulemahal I haven't seen this before, no. I'll look into it.

**Pull request merged**